qa/standalone/osd/divergent-priors.sh

```diff
@@ -153,7 +150,10 @@ function TEST_divergent() {
     objname="existing_$(expr $DIVERGENT_WRITE + $DIVERGENT_REMOVE)"
     echo "writing non-divergent object $objname"
     ceph pg dump pgs
-    rados -p $poolname put $objname $dummyfile2
+    # a second object (using a different size, for good measure)
+    local dummyfile2=$(file_with_random_data 1000)
+    rados -p $poolname put $objname $dummyfile2 || return 1
+    rm -f $dummyfile2
```

Comment:
since this test is not evaluating the handling of an actual input file, you can do this more cleanly:

```
dd if=/dev/urandom bs=1000 count=1 | rados -p "$poolname" put "$objname" -
```

Reply:
(and elsewhere)

(( this is a nit ))

Reply:
Will change in all tests where dummyfile2 is used exactly once (as otherwise I'll have to verify that the test is not based on the assumption that the data is the same)